### PR TITLE
Add search field to django admin

### PR DESCRIPTION
**Pull request by szok**
Added search fields to model admin: SoftwareCategory, LicenceType, Warehouse,
  ServiceEnvironment, Service, Environment

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/assets/admin.py [hunk 1]
```diff
@@ -47,14 +51,15 @@ class AssetModelAdmin(
 ):
 
     resource_class = resources.AssetModelResource
-
     list_select_related = ['manufacturer']
     raw_id_fields = ['manufacturer']
+    search_fields = ['name']
```
Comment (mkurek) [inline src/ralph/assets/admin.py]: add searching by manufacturer name too

Comment (szok) [inline src/ralph/assets/admin.py]: Done

**Comment by mkurek** [comment]
:ok:

**Pull request merged**